### PR TITLE
Completed HW 30.1

### PR DESCRIPTION
**Pull request by RomanMinevich**

### CODE REVIEW

File: src/main/java/mate/academy/spring/MainApp.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+import java.sql.SQLException;
+import java.util.List;
+import mate.academy.spring.config.AppConfig;
+import mate.academy.spring.entity.User;
+import mate.academy.spring.service.UserService;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+public class MainApp {
+    public static void main(String[] args) throws SQLException {
```
Comment (Sofasmile) [inline src/main/java/mate/academy/spring/MainApp.java]: add checking for book

File: src/main/java/mate/academy/spring/config/AppConfig.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    public LocalSessionFactoryBean getSessionFactory() {
+        LocalSessionFactoryBean factoryBean = new LocalSessionFactoryBean();
+        factoryBean.setDataSource(getDataSource());
+
+        Properties props = new Properties();
+        props.put("hibernate.show_sql", env.getProperty("hibernate.show_sql"));
+        props.put("hibernate.hbm2ddl.auto", env.getProperty("hibernate.hbm2ddl.auto"));
+
+        factoryBean.setHibernateProperties(props);
+        factoryBean.setAnnotatedClasses(User.class);
```
Comment (Sofasmile) [inline src/main/java/mate/academy/spring/config/AppConfig.java]:
```suggestion
        factoryBean.setAnnotatedClasses(User.class, Book.class);
```
and remove next line. or use package scanning

File: src/main/java/mate/academy/spring/entity/Book.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "books")
+public class Book {
+    @Id
+    @GeneratedValue(strategy = IDENTITY)
+    Long id;
```
Comment (Sofasmile) [inline src/main/java/mate/academy/spring/entity/Book.java]: add access modifier for all fields
remember about oop

File: src/main/java/mate/academy/spring/service/UserServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    public void add(User user) {
+        userDao.add(user);
+    }
+
+    @Transactional(readOnly = true)
+    @Override
+    public List<User> listUsers() {
+        return userDao.listUsers();
+    }
+
```
Comment (Sofasmile) [inline src/main/java/mate/academy/spring/service/UserServiceImpl.java]: remove empty line

File: src/main/java/mate/academy/spring/MainApp.java (hunk 1 above)
Comment (RomanMinevich) [reply src/main/java/mate/academy/spring/MainApp.java]: Added

File: src/main/java/mate/academy/spring/config/AppConfig.java (hunk 2 above)
Comment (RomanMinevich) [reply src/main/java/mate/academy/spring/config/AppConfig.java]: Fixed

File: src/main/java/mate/academy/spring/entity/Book.java (hunk 3 above)
Comment (RomanMinevich) [reply src/main/java/mate/academy/spring/entity/Book.java]: Fixed

File: src/main/java/mate/academy/spring/service/UserServiceImpl.java (hunk 4 above)
Comment (RomanMinevich) [reply src/main/java/mate/academy/spring/service/UserServiceImpl.java]: Done

**Pull request merged**